### ISSUE TITLE
Keyboard Modifer Request

### ISSUE THREAD

**Issue by BestLeaks** [SH42913/GunshotWound2#14]
Could you potentially add a Modifier Key Option?

**Comment by SH42913** [comment]
Yeah, I've already added that for one new feature. I just need to adapt old keys to support modifiers. There will be possible Shift/Alt/Ctrl.

**Comment by SH42913** [comment]
Also, thank you for issue creating :)

**Comment by BestLeaks** [comment]
perfect, furthermore I have a couple questions, hopefully you don't mind if I ask here as I don't want to bombard you with issues for each question. I am using you're updated pre-release you posted on GTA5 Mods

1. Is there a way to stop the effects (I seen in a YouTube video where the person had zero effects, but I can't seem to fully understand how the mod handles these)

2. I know this isn't weapon related but I've noticed if I get hit by a car or something it is also taking that into account, could crashes count?

that's all I've got for now, and again, thanks for this mod. I've been personally wanting something like this for so long then I found it 👍 I'll keep testing and if I catch anything, or find anything that could be worth adding, I'll open a new issue

Edit: I want to point out one thing that occurred to me, but I have no idea if I can again re-create it. there was several times where (I think) after I got shot while in a vehicle or something, I got out, and stayed ragdolled on the ground, regardless of if I had pain or not. I eventually got up but I have no idea how it occurred, if it happens again I'll grab the logs and the reproduce steps.

**Comment by SH42913** [comment]
> Is there a way to stop the effects

Just heal yourself in any way: use cheat, heal-key(GSW2Config.xml) or just take the closest first aid kit.

> could crashes count
 
 It is already in latest test-version. Crashes and falls are supported by GSW2.

> thanks for this mod

You're welcome

> if I catch anything, or find anything that could be worth adding, I'll open a new issue

Thank you so much :)

> regardless of if I had pain or not

Sounds strange, looks like there's small bug with that, it would be cool if you found way to reproduce

> I'll grab the logs and the reproduce steps

Use Debug version to have debug logs from GSW

**Comment by SH42913** [comment]
Will be added in 1.0 release, you may check release candidate in [this discord section](https://discordapp.com/channels/1263842381321732147/1264536701351759964)
